tox.ini

```diff
@@ -51,18 +56,62 @@ passenv =
     CI
     GITHUB*
 
+[flake8_plugins]
+# This is a pseudo-section that feeds into [testenv:qa] and GA
+# Snippets of letters above these plugins are tests that need to be "select"-ed in flake8 config (in
+# setup.cfg) to activate the respective plugins. If no snippet is given, that means the plugin is
+# always active.
+deps =
+    flake8-bugbear
+    flake8-builtins
+    flake8-coding
+    # C4
+    flake8-comprehensions
+    # JS
+    flake8-multiline-containers
+    # PIE
+    flake8-pie
+    # MOD
+    flake8-printf-formatting
+    # PT
+    flake8-pytest-style
+    # SIM
+    flake8-simplify
+    # Cognitive Complexity looks like a good idea, but to fix the complaints... it will be an epic effort.
+    # So we disable it for now and reenable when we're ready, probably just before 2.0
+    # # CCR
+    # flake8-cognitive-complexity
+    # ECE
+    flake8-expression-complexity
+    # C801
+    flake8-copyright
+    # DUO
+    dlint
+    # TAE
+    flake8-annotations-complexity
+    # TAE
+    flake8-annotations-coverage
+    # ANN
+    flake8-annotations
+    # YTT
+    flake8-2020
+    # N400
+    flake8-broken-line
```

Comment:
Extracted into a separate config section for easier reuse. Most notably the extraction code in the GA yml.